README.md

```diff
@@ -1,12 +1,248 @@
-# ICGC - Software Engineering Blog
+<!--Sources-->
+<!--https://blog.hubspot.com/marketing/what-is-a-pillar-page-->
+<!--https://thewritepractice.s3-us-west-1.amazonaws.com/Blog+Post+Template+Worksheet.pdf-->
+<!--https://www.verblio.com/blog/blog-post-length#:~:text=The%20ideal%20paragraph%20size%20for,that%20number%20can%20fluctuate%20wildly.-->
+<!--https://thewritepractice.com/formatting-tips/.-->
+<!--https://blog.hubspot.com/marketing/how-long-should-your-blog-posts-be-faq-->
+<!--https://www.wix.com/blog/2021/06/how-long-should-a-blog-post-be/#:~:text=After%20in%2Ddepth%20industry%20research,sweet%20spot%20right%20at%202%2C450.-->
 
-Although not mandatory, you might want to run a local instance of the blog to verify that your blog post content renders properly.
+# :newspaper: OICR Engineering Blog 
 
-## Instructions to run a local instance of Jekyll
+<img src="images/OICRHero.jpg">
 
-First clone this repo
+<br />
 
-Then cd into the repo and run:
+<p align="center"><a href="#book-writing-guide">Writing Guide</a>  |  <a href=#computer-running-a-local-instance>Running a Local Instance</a>  |  <a href=#ballot_box-submitting-a-blog-post>Submitting a Blog Post</a>
+
+We are the OICR team of software engineers, infrastructure professionals, and bioinformaticians developing big data tools to help in the fight against cancer. Here we share our successes, failures, curiosities and encouragement for open technical discussion with you, the reader. Welcome!
+
+<br />
+
+# :date: Bi-weekly Content Calendar 
+
+|Post Date| Author(s) |Topic|Reviewer|
+|---|---|---|---|
+|Dec 13|Ann Catton|```link google doc```|
+|Dec 27|Daniel Lu|```link google doc```|
+|Jan 10|Brandon Chan|```link google doc```|
+|Jan 24|Yelizar Alturmessov|```link google doc```|
+|Feb 7|Edmund Su|```link google doc```|
+|Feb 21|Jon Eubank|```link google doc```|
+|Mar 7|Samantha Rich|```link google doc```|
+|Mar 21|Alexis Li|```link google doc```|
+|Apr 4|Daniel DeMaria|```link google doc```|
+|Apr 18|Mitchell Shiell|```link google doc```|
+|May 2|Linda Xiang|```link google doc```|
+|May 16|Ummulkiram Rangwala|```link google doc```|
+|May 30|Justin Richardsson|```link google doc```|
+|June 14|Rakesh Mistry|```link google doc```|
+
+<br />
+
+# :book: Writing Guide
```

Comment:
can we move the writing guide to a separate .md file, leaving the main README with the typical content: overview of project, link to live site,  how to interact with the repo (running locally, links to writing guide and topic suggestions, etc.)

Reply:
we can link to it from the main file of course...

Reply:
Writing guided moved to a separate file, and topic suggestions were linked on the main readme in the latest commit